cylc/flow/task_events_mgr.py

```diff
@@ -790,7 +790,7 @@ def process_message(
             self.workflow_db_mgr.put_update_task_jobs(
                 itask, {"run_signal": signal})
             if self._process_message_failed(
-                itask, event_time, self.JOB_FAILED, forced
+                itask, event_time, self.JOB_FAILED, forced, message
             ):
```

Comment:
I think you've missed a `_process_message_failed` somewhere which will also need the `message` argument added.